src/main/java/net/dries007/tfc/client/render/LoomBlockEntityRenderer.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed under the EUPL, Version 1.2.
+ * You may obtain a copy of the Licence at:
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
+
+package net.dries007.tfc.client.render;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.renderer.blockentity.BlockEntityRenderer;
+import net.minecraft.client.renderer.blockentity.BlockEntityRendererProvider;
+import net.minecraft.client.renderer.texture.TextureAtlas;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.blaze3d.vertex.VertexConsumer;
+import com.mojang.math.Matrix3f;
+import com.mojang.math.Matrix4f;
+import com.mojang.math.Vector3f;
+import net.dries007.tfc.client.ClientHelpers;
+import net.dries007.tfc.common.blockentities.LoomBlockEntity;
+import net.dries007.tfc.common.blocks.wood.TFCLoomBlock;
+
+public class LoomBlockEntityRenderer implements BlockEntityRenderer<LoomBlockEntity>
+{
+    public LoomBlockEntityRenderer(BlockEntityRendererProvider.Context context)
```

Comment:
I think you can register the renderer like `ctx -> new LoomBER()` and skip having a constructor entirely

Reply:
Turns out that was just a remainder from porting it from latejuly and isn't needed currently, though I can change that line still if you want to match the current style